commit 56f3b3696b611c0f9aa0f3d41a3f8a29ba5960ee
Author: Benjamin Herrenschmidt <benh@kernel.crashing.org>
Date:   Wed Aug 16 16:01:14 2017 +1000

    powerpc: Fix VSX enabling/flushing to also test MSR_FP and MSR_VEC
    
    commit 5a69aec945d27e78abac9fd032533d3aaebf7c1e upstream.
    
    VSX uses a combination of the old vector registers, the old FP
    registers and new "second halves" of the FP registers.
    
    Thus when we need to see the VSX state in the thread struct
    (flush_vsx_to_thread()) or when we'll use the VSX in the kernel
    (enable_kernel_vsx()) we need to ensure they are all flushed into
    the thread struct if either of them is individually enabled.
    
    Unfortunately we only tested if the whole VSX was enabled, not if they
    were individually enabled.
    
    Fixes: 72cd7b44bc99 ("powerpc: Uncomment and make enable_kernel_vsx() routine available")
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
    Signed-off-by: Michael Ellerman <mpe@ellerman.id.au>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1648_56f3b3696b611c0f9aa0f3d41a3f8a29ba5960ee.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1648_56f3b3696b611c0f9aa0f3d41a3f8a29ba5960ee.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
